### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by rvanheest**
Some remarks on the setup of `narcis-graphql`. Also fixed a bug with parsing the variables, discovered earlier this week in `easy-deposit-properties`.

@PaulBoon no substantial changes, just the setup...
The substantial things will come on Monday.

@DANS-KNAW/narcis for review

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/narcis/graphql/Command.scala [hunk 1]
```diff
@@ -60,6 +60,7 @@ object Command extends App with DebugEnhancedLogging {
     val service = new NarcisGraphqlService(configuration.serverPort, Map(
       "/" -> new NarcisGraphqlRootServlet(configuration.version),
       "/graphql" -> new GraphQLServlet(configuration.profilingThreshold, repository.repository),
+      "/graphiql" -> new GraphiQLServlet("/graphql"),
```
Comment (wilkos-dans) [inline src/main/scala/nl/knaw/dans/narcis/graphql/Command.scala:63]: Is it common practice to include a client into the server?

Comment (rvanheest) [reply src/main/scala/nl/knaw/dans/narcis/graphql/Command.scala:63]: (1) in most of the projects I've seen (mostly demo projects) on GitHub, a client for experimenting is included
(2) I think it's at least useful to have a dedicated client for GraphQL. It mainly allows you to (a) experiment with the data and (b) browse through the API and documentation quickly.
(3) That is not to say that you cannot have other solutions, but these require extra installation(s). A dedicated client in your gets automatically installed with the server.

Comment (PaulBoon) [reply src/main/scala/nl/knaw/dans/narcis/graphql/Command.scala:63]: We can use the GraphiQL standalone application or even Postman. 
It seems nice to have this 'builtin' client, but only if it gives minimal maintenance, so of the shelf libs etc.

**Comment by PaulBoon** [comment]
Looks allright to me

**Pull request merged**